packages/react-data-grid/style/rdg-header.less

```diff
@@ -38,97 +22,93 @@
+.rdg-cell .Select {
     max-height: 30px;
     font-size: 12px;
     font-weight: normal;
 }
 
-.react-grid-HeaderCell .Select-control {
+.rdg-cell .Select-control {
     max-height: 30px;
     border: 1px solid #cccccc;
     color: #555;
     border-radius: 3px;
 }
 
-.react-grid-HeaderCell .is-focused:not(.is-open) > .Select-control {
+.rdg-cell .is-focused:not(.is-open) > .Select-control {
     border-color: #66afe9;
     outline: 0;
     -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(102,175,233,.6);
     box-shadow: inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(102,175,233,.6);
 }
 
-.react-grid-HeaderCell .Select-control .Select-placeholder {
+.rdg-cell .Select-control .Select-placeholder {
     line-height: 20px;
     color: #999;
     padding: 4px;
 }
 
-.react-grid-HeaderCell .Select-control .Select-input {
+.rdg-cell .Select-control .Select-input {
     max-height: 28px;
     padding: 4px;
     margin-left: 0px
 }
 
-.react-grid-HeaderCell .Select-control .Select-input input {
+.rdg-cell .Select-control .Select-input input {
     padding: 0px;
     height: 100%;
 }
 
-.react-grid-HeaderCell .Select-control .Select-arrow-zone .Select-arrow {
+.rdg-cell .Select-control .Select-arrow-zone .Select-arrow {
     border-color: gray transparent transparent;
     border-width: 4px 4px 2.5px;
 }
 
 
-.react-grid-HeaderCell .Select-control .Select-value {
+.rdg-cell .Select-control .Select-value {
     padding: 4px;
     line-height: 20px !important;
 }
 
-.react-grid-HeaderCell .Select--multi .Select-control .Select-value {
+.rdg-cell .Select--multi .Select-control .Select-value {
     padding: 0px;
     line-height: 16px !important;
     max-height: 20px;
 }
 
-.react-grid-HeaderCell .Select--multi .Select-control .Select-value .Select-value-icon {
+.rdg-cell .Select--multi .Select-control .Select-value .Select-value-icon {
     max-height: 20px;
 }
 
-.react-grid-HeaderCell .Select--multi .Select-control .Select-value .Select-value-label {
+.rdg-cell .Select--multi .Select-control .Select-value .Select-value-label {
     max-height: 20px;
 }
 
-.react-grid-HeaderCell .Select-control .Select-value .Select-value-label {
+.rdg-cell .Select-control .Select-value .Select-value-label {
     color: #555555 !important;
 }
 
-.react-grid-HeaderCell .Select-menu-outer {
+.rdg-cell .Select-menu-outer {
   z-index: 2;
 }
 
-.react-grid-HeaderCell .Select-menu-outer .Select-option {
+.rdg-cell .Select-menu-outer .Select-option {
     padding: 4px;
     line-height: 20px;
 }
 
-.react-grid-HeaderCell .Select-menu-outer .Select-menu .Select-option.is-selected {
+.rdg-cell .Select-menu-outer .Select-menu .Select-option.is-selected {
     color: #555555;
 }
 
-.react-grid-HeaderCell .Select-menu-outer .Select-menu .Select-option.is-focused {
+.rdg-cell .Select-menu-outer .Select-menu .Select-option.is-focused {
     color: #555555;
 }
 
 .rdg-sortable-header-cell {
   cursor: pointer;
 }
+
+.rdg-header-sort {
```

Comment:
maybe using `flex` and put 2 `div`s inside rather than just `float: right`?

Reply:
It just works though?

Reply:
My concern is that when the columns width is `100px` while the header text width is like `200px`, will the sort icon being pushed to next line or keep in the view. I believe using `float: right` will push it to next line. I am trying to run this PR in my local but having some packages installing issue.